website/docs/hail-format.md

```diff
@@ -48,7 +48,9 @@ from ukbb_pan_ancestry import *
 hl.init(spark_conf={'spark.hadoop.fs.gs.requester.pays.mode': 'AUTO',
                     'spark.hadoop.fs.gs.requester.pays.project.id': 'your_project_id'})
 
-mt = load_final_sumstats_mt()
+# to load all results for which sumstats were exported (lambda GC < 0.5 or > 5)
```

Comment:
Was this supposed to have some code associated?

Reply:
Made this a bit clearer